File: week_06/day_3/composition/Composition_Start/app/src/test/java/example/codeclan/com/composition_start/MopTest.java
package example.codeclan.com.composition_start;

/**
 * Created by user on 28/08/2017.
 */

import static org.junit.Assert.*;
import org.junit.*;

import example.codeclan.com.composition_start.wizard_management.*;


public class MopTest {

    Mop mop;

    @Before
    public void before(){
        mop = new Mop("Hygenic");
    }

    @Test
    public void hasBrand(){
        assertEquals("Hygenic", mop.getBrand());
    }

}

File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management_specs/MopTest.cs
using NUnit.Framework;

namespace WizardManagement { 

  [TestFixture]
  public class MopTest {
    Mop mop;

    [SetUp]
    public void Init() 
    {
      mop = new Mop("Hygenic");
    }

    [Test]
    public void HasBrand()
    {
      Assert.AreEqual("Hygenic", mop.Brand);
    }

  }
}